### DISCUSSION TITLE
CSSBattle's LEARN - Level #77

### DISCUSSION THREAD

**Discussion by a5ena** [cssbattle/forum#308] [General]
Hello folks!

I've been trying to complete level #77 of CSSBattle's LEARN - it's one task before the goal and I'm stuck.
I think I have the right solution. Shouldn't be that hard... 😅

![_2022-09-22_1654__2022CBD01__snipping__level_78__bea](https://user-images.githubusercontent.com/73748665/191781734-4d6e1c5d-83c7-458f-a762-a19e713250c5.PNG)

I have tried severel times...
What did I miss?

You can have a look at
https://cssbattle.dev/learn/77

Maybe you have an idea!

Thank you very much for your reply!

**Comment by aarthvaen** [comment]
The selector order should be correct, but it's applied to the wrong line of code.

Below `.button` and `.loader` you should have a `selector` pseudo-element with the `border-color: darkblue;` property. Try replacing the `selector` word with `.dark-theme .button .loader` selector order that you have.
